src/c++/vdo/user/messageStatsReader.c

```diff
@@ -0,0 +1,1272 @@
+// SPDX-License-Identifier: GPL-2.0-only
+/*
+ * Copyright 2023 Red Hat
+ *
+ * ATTENTION!!! ATTENTION!!! ATTENTION!!! ATTENTION!!! ATTENTION!!!
+ *
+ * If you add new statistics, be sure to update the following files:
+ *
+ * ../../../c++/vdo/base/statistics.h
+ * ../../../c++/vdo/base/message-stats.c
+ * ../../../c++/vdo/base/pool-sysfs-stats.c
+ * ../../../c++/vdo/user/vdoStatsWriter.c
+ * ../../../perl/Permabit/Statistics/Definitions.pm
+ *
+ */
```

Comment:
Similar comment on shortening these paths. Alnd please remove the ATTENTION line and any unnecessary blank lines (like line 14).